py/fastspecfit/emlines.py

```diff
@@ -468,7 +468,7 @@ def _fix_parameters(linemodel, verbose=False):
                 [O2] (5-->3): airwave: 7329.6613 vacwave: 7331.6807 emissivity: 1.35614e-23
                 [O2] (4-->3): airwave: 7330.7308 vacwave: 7332.7506 emissivity: 1.27488e-23
                 """
-                final_linemodel['tiedfactor'][param_names == linename+'_amp'] = 1.0 / 1.2251
+                final_linemodel['tiedfactor'][param_names == linename+'_amp'] = 1.2251
                 final_linemodel['tiedtoparam'][param_names == linename+'_amp'] = np.where(param_names == 'oii_7330_amp')[0]
```

Comment:
Instead of changing the line-ratio, can you please change the line which is fitted (7320, the stronger line) and the line which that line is tied to (7330, the weaker line, with an amplitude of 1/1.2251). The reason is that only the "untied" lines are modeled, so we want to throw the *stronger* of the doublet lines to the optimizer.